examples/vadd/vaddhost.f90

```diff
@@ -0,0 +1,95 @@
+
+#ifdef JIT
+#define FILENAME "vadddevice.cl"
+#elif INTEL_FPGA
+#define FILENAME "vadddevice.aocx"
+#elif XILINX_FPGA
+#define FILENAME "vadddevice.xclbin"
+#endif
+
+program vadd
+  use fortcl
+  use clfortran
+  use ocl_utils_mod, only: check_status
+  use iso_c_binding
+  implicit none
+
+  integer, parameter :: N = 100
+  integer(c_intptr_t), target :: deviceA, deviceB, deviceC
+  integer(c_intptr_t), target :: vaddkernel
+  integer(c_intptr_t), pointer :: cmd_queues(:)
+  integer(c_size_t), target :: globalsize(1) = (/N/)
+  real, dimension(N), target :: hostA, hostB, hostC, AfromDevice
+
+  integer(c_size_t) :: arraysize = c_sizeof(hostA)
+  integer(c_size_t), parameter :: zero = 0
+  integer :: i, ierr
+
+  ! Initialize B and C with random numbers
+  call RANDOM_NUMBER(hostB)
+  call RANDOM_NUMBER(hostC)
+
+  ! Perform host vector add
+  hostA = hostB + hostC
+
+
+  ! Initialise the OpenCL device and environment
+  call ocl_env_init()
+  cmd_queues => get_cmd_queues()
+
+  ! Allocate read-write buffers on the device
+  deviceA = create_rw_buffer(arraysize)
+  deviceB = create_rw_buffer(arraysize)
+  deviceC = create_rw_buffer(arraysize)
+
+  ! Load a kernel (expects pre-compiled kernels)
```

Comment:
Please correct this comment (e.g. if suffix is ".cl" then kernel will be compiled at run-time).

Reply:
Fixed